payments/tests/test_payments_talpa_product_accounting.py

```diff
@@ -0,0 +1,88 @@
+from decimal import Decimal
+
+import pytest  # noqa
+
+from leases.enums import LeaseStatus
+from leases.models import BerthLease
+from payments.enums import OrderStatus, OrderType, ProductServiceType, TalpaProductType
+from payments.exceptions import TalpaProductAccountingNotFoundError
+from payments.models import TalpaProductAccounting
+from payments.tests.conftest import *  # noqa
+from payments.tests.factories import OrderFactory, OrderLineFactory
+from resources.enums import AreaRegion
+
+
+@pytest.mark.parametrize(
+    "order_with_products",
+    ["berth_order", "winter_storage_order"],
+    indirect=True,
+)
+@pytest.mark.parametrize("region", [AreaRegion.WEST, AreaRegion.EAST])
+def test_berth_product_order(
```

Comment:
To be more consistent with other tests' naming, I guess the name could include also "winter storage"?

Reply:
Or just plain `product` order :derp: but nice idea